Streetcode/DbUpdate/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿using DbUp;
+using Microsoft.Extensions.Configuration;
+
+public class Program
+{
+    static int Main(string[] args)
+    {
+        string seedingPath = Directory.GetCurrentDirectory() + @"\Streetcode.DAL\Persistence\ScriptsMigration\";
+        string migrationPath = Directory.GetCurrentDirectory() + @"\Streetcode.DAL\Persistence\ScriptsSeeding\";
+
+        var environment = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") ?? "Local";
+
+        var configuration = new ConfigurationBuilder()
+            .SetBasePath(Directory.GetCurrentDirectory() + @"\Streetcode.WebApi")
+            .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
+            .AddJsonFile($"appsettings.{environment}.json", optional: true, reloadOnChange: true)
+            .AddEnvironmentVariables("STREETCODE_")
+            .Build();
+
+        var connectionString = configuration.GetConnectionString("DefaultConnection");
+
+        string pathToScript = "";
+
+        while (pathToScript != "1" && pathToScript != "2")
+        {
+            Console.WriteLine("Enter '1' to MIGRATE or '2' to SEED db:");
+            pathToScript = Console.ReadLine();
+        }
```

Comment:
let's move this to args, so if we pass something like "-s or -seeding" we will seed database, if "-m or -migrate" we will migrate only. If this is not provided let's set default to migrate

Reply:
got it!